### DISCUSSION TITLE
Vulnerability in the quorum version 22.7.2

### DISCUSSION THREAD

**Discussion by bipeen-kumar** [ConsenSys/quorum#1552] [Q&A] [answered]
Hi Team

There is a vulnerability for github.com/gogo/protobuf v1.3.1 in the latest quorum version `22.7.2`
Please could you let us know your plan to fix this vulnerability for `github.com/gogo/protobuf v1.3.1` and upgrade it to `1.3.2`?

**Comment by baptiste-b-pegasys** [comment]
Hello I am sorry I don't find this direct dependency in the go.mod file.
github.com/gogo/protobuf v1.3.1
We have
github.com/golang/protobuf v1.5.2

**Comment by antonydenyer** [comment, marked as answer]
fixed with https://github.com/ConsenSys/quorum/pull/1571
